conjur/api/client.py

```diff
@@ -99,9 +97,10 @@ def __init__(self,
                               f"'cert_file': {loaded_config['ca_bundle']}}}")
             except CertificateVerificationException:
                 raise
-            # TODO add error handling for when conjurrc field doesn't exist
             except Exception as exc:
-                raise ConfigException(exc) from exc
+                raise ConfigurationMissingException("The conjurrc configuration file is either missing "
+                                                    "or missing parameters. Ensure that it exists and contains "
```

Comment:
@InbalZilberman need you 👍  here

Reply:
@sigalsax How does the user make sure it exists? how about asking the user to run conjur init if needed?
has TW reviewed it?

Reply:
reached out to Shuli.

The user makes it exist by reinitializing the client. I updated the message to
"The conjurrc configuration file is either missing, empty, or missing parameters. Reinitialize the client or ensure that the conjurrc contains 'conjur_account', 'conjur_url', and 'cert_file'."